src/components/vehicleReference/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, ReactNode } from 'react';
+
+import {
+  AspectRatio,
+  Box,
+  chakra,
+  useMultiStyleConfig,
+} from '@chakra-ui/react';
+
+import Stack from '../stack';
+
+interface Props {
+  renderImage: ReactNode;
+  carTitle: string;
```

Comment:
How about just
```suggestion
  title: string;
```
?
(or `vehicleTitle` if we want to be more specific)

Reply:
Nice, I will rename it `vehicleTitle`. Thanks!